rasa/core/processor.py

```diff
@@ -125,25 +125,25 @@ async def handle_message(
         tracker = await self.log_message(message, should_save_tracker=False)
 
         if self.model_metadata.training_type == TrainingType.NLU:
-            self._save_tracker(tracker)
+            self.save_tracker(tracker)
             rasa.shared.utils.io.raise_warning(
                 "No core model. Skipping action prediction and execution.",
                 docs=DOCS_URL_POLICIES,
             )
             return None
 
-        tracker = await self._run_action_extract_slots(message.output_channel, tracker)
+        tracker = await self.run_action_extract_slots(message.output_channel, tracker)
 
         await self._run_prediction_loop(message.output_channel, tracker)
 
-        self._save_tracker(tracker)
+        self.save_tracker(tracker)
 
         if isinstance(message.output_channel, CollectingOutputChannel):
             return message.output_channel.messages
 
         return None
 
-    async def _run_action_extract_slots(
+    async def run_action_extract_slots(
         self, output_channel: OutputChannel, tracker: DialogueStateTracker,
     ) -> DialogueStateTracker:
```

Comment:
I'm going to predict code quality will fail because of lack of docstrings, both here and in `save_tracker`.

Reply:
ouch, yeah, should've seen that, thanks :D